src/getting-started/starting-stack.md

```diff
@@ -127,94 +119,4 @@ Stopping the local validator...
 Local validator stopped successfully!
 ```
 
```

Comment:
Maybe we should mention that they can start a full local network of validators but then just reference the README for arch-cli instead of documenting it all out here? What do you think?

Reply:
Yeah, I can add that so at least it's still documented as a possibility

Reply:
[0346c41](https://github.com/Arch-Network/book/pull/13/commits/0346c41e70fc6253bdebf281a5c983371e4d52e5)